manic/repository.py

```diff
@@ -30,6 +36,14 @@ def __init__(self, component_name, repo):
                 (self._hash is EMPTY_STR)):
             fatal_error('{0} repo must have a branch, tag or hash element')
 
+        if self._user_name.lower() == 'GIT_AUTHOR_NAME'.lower():
+            # check GIT_AUTHOR_NAME environment variable
+            git_author_name = os.getenv('GIT_AUTHOR_NAME')
+            if not git_author_name is None:
```

Comment:
I believe `XXX is not` is preferred to `not XXX is` (see https://www.python.org/dev/peps/pep-0008/#programming-recommendations)